src/components/MapDisplay/MapDisplay.js

```diff
@@ -1,13 +1,44 @@
 import React, { Component } from 'react'
+
+
+import image from './mapPin.png'
 import './MapDisplay.css'
 
+
 class MapDisplay extends Component {
     
+    componentDidMount () {
+        //https://leafletjs.com/examples/custom-icons/
+        const sanJoseLongitude = -121.8863
+        const sanJoseLatitude = 37.3382
```

Comment:
Put this in config in case other cities want to use this?

Reply:
for a later feature